index.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(options) {
             return cb(new gutil.PluginError('gulp-image-optimization', 'Streaming not supported'));
         }
 
-        if (['.jpg', '.jpeg', '.png', '.gif'].indexOf(path.extname(file.path)) === -1) {
+        if (['.jpg', '.JPG', '.JPEG', '.jpeg', '.png', '.PNG', '.gif', '.GIF'].indexOf(path.extname(file.path)) === -1) {
```

Comment:
I'm going to put the extension to lowercase before comparing thanks for the feedback! I'll get this done this weekend. 

Cheers!